### PR TITLE
feat(apple): Troubleshooting "Failed to write data to path"

### PR DESCRIPTION
**Pull request by brustolin**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Adding an explanation on what to do if the user sees this error: `[SentryFileManager:439] Failed to write data to path`

### CODE REVIEW

File: src/platforms/apple/troubleshooting.mdx [hunk 1]
```diff
@@ -35,3 +35,9 @@ If you use Tuist to manage your SPM dependencies, add the following configuratio
 ```Swift
 "Sentry": ["USE_HEADERMAP": "YES"]
 ```
+
+## SentryFileManager: Failed to write data to path
+
+Sentry needs to write files in a specific path in the app cache directory. This path is created during the SDK initialization.
+If you see a log that looks like this: `[SentryFileManager:439] Failed to write data to path` it means that the cache path don`t exist anymore.
```
Comment (kahest) [inline src/platforms/apple/troubleshooting.mdx:42]:
```suggestion
If you see a log message that looks like this: `[SentryFileManager:439] Failed to write data to path` it means that the cache path doesn't exist anymore.
```

File: src/platforms/apple/troubleshooting.mdx [hunk 2]
```diff
@@ -35,3 +35,9 @@ If you use Tuist to manage your SPM dependencies, add the following configuratio
 ```Swift
 "Sentry": ["USE_HEADERMAP": "YES"]
 ```
+
+## SentryFileManager: Failed to write data to path
+
+Sentry needs to write files in a specific path in the app cache directory. This path is created during the SDK initialization.
```
Comment (kahest) [inline src/platforms/apple/troubleshooting.mdx:41]:
```suggestion
Sentry writes and reads files at a specific path in the app cache directory. This path is created during the SDK initialization.
```

File: src/platforms/apple/troubleshooting.mdx [hunk 3]
```diff
@@ -35,3 +35,9 @@ If you use Tuist to manage your SPM dependencies, add the following configuratio
...
+If you see a log that looks like this: `[SentryFileManager:439] Failed to write data to path` it means that the cache path don`t exist anymore.
+Check your app for any procedure that cleans the cache of your app during runtime, and avoid deleting Sentry directory.
```
Comment (kahest) [inline src/platforms/apple/troubleshooting.mdx:43]: `h`: How to avoid this? Would need to know what the path is called and exclude that?

**Review by kahest — CHANGES_REQUESTED** [review]
Thanks for adding this! We should try to give more information on how to actually solve this

File: src/platforms/apple/troubleshooting.mdx (hunk 3 above)
Comment (brustolin) [reply src/platforms/apple/troubleshooting.mdx:43]:
```suggestion
Check your app for any procedure that cleans the cache of your app during runtime, and avoid deleting `io.sentry` directory.
```

File: src/platforms/apple/troubleshooting.mdx [hunk 4]
```diff
@@ -35,3 +35,9 @@ If you use Tuist to manage your SPM dependencies, add the following configuratio
 ```Swift
 "Sentry": ["USE_HEADERMAP": "YES"]
 ```
+
+## SentryFileManager: Failed to write data to path
+
+Sentry writes and reads files at a specific path in the app cache directory. This path is created during the SDK initialization.
+If you see a log message that looks like this: `[SentryFileManager:439] Failed to write data to path` it means that the cache path doesn't exist anymore.
+Check your app for any procedure that cleans the cache of your app during runtime, and avoid deleting `io.sentry` directory.
```
Comment (kahest) [inline src/platforms/apple/troubleshooting.mdx:43]:
```suggestion
Check your app for any procedure that cleans the cache of your app during runtime, and avoid deleting the `io.sentry` directory.
```

**Review by kahest — APPROVED** [review]
One last wording change, then LGTM

**Review by philipphofmann — APPROVED** [review]
LGTM

**Pull request merged**